pkgs/development/libraries/harfbuzz/default.nix

```diff
@@ -33,19 +38,23 @@ stdenv.mkDerivation {
   outputs = [ "out" "dev" ];
   outputBin = "dev";
 
-  configureFlags = [
-    # not auto-detected by default
-    "--with-graphite2=${if withGraphite2 then "yes" else "no"}"
-    "--with-icu=${if withIcu then "yes" else "no"}"
-    "--with-gobject=yes"
-    "--enable-introspection=yes"
-  ]
-    ++ stdenv.lib.optional withCoreText "--with-coretext=yes";
+  mesonFlags = [
+    (boolMesonFlag "graphite" withGraphite2)
+    (boolMesonFlag "icu" withIcu)
+    (boolMesonFlag "coretext" withCoreText)
+    (boolMesonFlag "gobject" true)
+    (boolMesonFlag "introspection" true)
+    (boolMesonFlag "tests" true)
+    # TODO build docs with gtk-docs without it trying to access the network
+    (boolMesonFlag "docs" false)
```

Comment:
That would require `gtk-doc` and `docbook-xsl-nons` dependencies. Also add `devdoc` to `outputs`.

Reply:
Main problem is that gtkdocs-scan tries to fetch stuff from network, so I think building them using nix requires some patching, not sure if it's worth it.

Reply:
It should only try to fetch docbook stylesheets and DTDs and if you add `docbook-xsl-nons`, it will find them in the catalogue.

Reply:
Still tries to fetch something and outputs broken includes, should investigate.

Reply:
Do you have the error message?

Reply:
[snippet from the build log](https://gist.github.com/sternenseemann/cfe8d5db085af423e53fc73523499ed3).

Outputs pages to `devdoc`, but many of them have this instead of their content:

```
 <xi:include><xi:fallback></xi:fallback></xi:include>
```

Reply:
`docbook_xml_dtd_43` will fix the doctype request but not sure if that will help with the XInclude elements not being processed.

Reply:
That fixed everything, enabled documentation.